5.0/en/0x53-V53-WebRTC.md

```diff
@@ -35,7 +35,9 @@ These requirements only apply to systems that host TURN servers as part of their
 
 Media security is paramount in WebRTC applications, as it directly impacts the confidentiality, integrity and availability of audio and video communications. By addressing these security concerns, developers can safeguard the media streams in WebRTC applications, preventing eavesdropping, tampering, and denial-of-service attacks that could compromise user privacy and communication quality.
 
-These requirements only apply to systems that...
+These requirements only apply to systems that host their own WebRTC media servers, such as Selective Forwarding Units (SFUs), Multipoint Control Units (MCUs), recording servers, or gateway servers. These servers are responsible for handling, routing, processing, or storing media streams within the application. The security of these media servers is crucial because they manage and distribute media between peers, and improper security could lead to unauthorized access, interception, or manipulation of media streams.
+
+Systems that rely solely on peer-to-peer media communication between web browsers, without the involvement of intermediate media servers, are excluded from these specific media-related security requirements. However, such systems should still adhere to general WebRTC security best practices to protect the communication between peers.
```

Comment:
@sandrogauci when you say:

> such systems should still adhere to general WebRTC security best practices to protect the communication between peers.

Do you mean other requirements in this section or are you referring to something else? Could you make it clearer?